### PR TITLE
Create zh_CN.po

### PR DESCRIPTION
**Pull request by dongfengweixiao**
Add zh_CN.po for simplified Chinese users

### CODE REVIEW

File: po/zh_CN.po [hunk 1]
```diff
@@ -0,0 +1,552 @@
+# Copyright (C) 2016 Mario Cianciolo <mr.udda@gmail.com>
+# This file is distributed under the same license as the multiload-ng package.
+# Roberta Assennato <r.assennato@gmail.com>, 2016.
```
Comment (udda) [inline po/zh_CN.po]: There should be your name and email ( `Dee.H.Y <dongfengweixiao@hotmail.com>, 2016` )

File: po/zh_CN.po [hunk 2]
```diff
@@ -0,0 +1,552 @@
...
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: multiload-ng\n"
+"Report-Msgid-Bugs-To: \n"
```
Comment (udda) [inline po/zh_CN.po]: There should be your name and email ( `Dee.H.Y <dongfengweixiao@hotmail.com>, 2016` )

File: po/zh_CN.po [hunk 3]
```diff
@@ -0,0 +1,552 @@
...
+"POT-Creation-Date: 2016-09-09 21:58+0200\n"
+"PO-Revision-Date: 2016-09-17 15:33+0800\n"
+"Language-Team: French\n"
```
Comment (udda) [inline po/zh_CN.po]: Write here "Simplified Chinese" instead of "French"

File: po/zh_CN.po [hunk 4]
```diff
@@ -0,0 +1,552 @@
...
+msgid "s"
+msgstr "秒"
+
+#, c-format
+msgid "Unable to open the following url: '%s'"
+msgstr "无法打开以下网址：‘%s’"
+
+#. xgettext: Translator Full Name <translator_email@example.com>
+msgid "translator-credits"
+msgstr "translator-credits"
```
Comment (udda) [inline po/zh_CN.po]: There should be your name and email ( `Dee.H.Y <dongfengweixiao@hotmail.com>, 2016` ) in place of "translator-credits"

**Comment by dongfengweixiao** [comment]
has fixed

**Comment by udda** [comment]
Thank you!

**Pull request merged**